Integrations/SentinelOne-V2/SentinelOne-V2.py

```diff
@@ -1516,6 +1516,249 @@ def broadcast_message():
         return_error('No messages were sent. Verify that the inputs are correct.')
 
 
+def shutdown_agents_request(query, agent_id, group_id):
+
+    endpoint_url = 'agents/actions/shutdown'
+    filters = {}
+
+    if query:
+        filters['query'] = query
+    if agent_id:
+        filters['ids'] = agent_id
+    if group_id:
+        filters['groupIds'] = group_id
+
+    payload = {
+        "filter": filters,
+        "data": {}
+    }
+
+    response = http_request('POST', endpoint_url, data=json.dumps(payload))
+    if response.get('errors'):
+        return_error(response.get('errors'))
+    else:
+        return response
+
+
+def shutdown_agents():
+    """
+    Sends a shutdown command to all agents matching the input filter
+    """
+    query = demisto.args().get('query', '')
+
+    agent_id = argToList(demisto.args().get('agent_id'))
+    group_id = argToList(demisto.args().get('group_id'))
+
+    affected_agents = shutdown_agents_request(query, agent_id, group_id)
+    agents = affected_agents.get('data', {}).get('affected', 0)
+    if agents > 0:
+        contents = {
+            'ID': agent_id
+        }
+    else:
+        return_error('No agents were shutdown.')
+
+    context = {
```

Comment:
dont see really the point returning context here, unless we can add some more data other than the ID